packages/client/src/rpc/modules/eth.ts

```diff
@@ -385,10 +387,12 @@ export class Eth {
       const { execResult } = await vm.evm.runCall(runCallOpts)
       return bytesToHex(execResult.returnValue)
     } catch (error: any) {
-      throw {
-        code: INTERNAL_ERROR,
-        message: error.message.toString(),
+      const e: any = {
```

Comment:
This all looks great.  How hard is it to add a type to the `e` value that's thrown by each handler?

Reply:
Fixed